src/NetworkTestWidget/SettingsModal/SettingsModal.tsx

```diff
@@ -176,19 +185,19 @@ export default function SettingsModal({
             <Grid container direction="column">
               <FormControlLabel
                 control={<Radio checked={codec === Opus} onChange={handleCodecChange} name={Opus} />}
-                label="Opus"
+                label="opus"
               />
               <FormControlLabel
                 control={<Radio checked={codec === PCMU} onChange={handleCodecChange} name={PCMU} />}
-                label="PCMU"
+                label="pcmu"
               />
               <FormControlLabel
                 control={<Radio checked={codec === Opus + PCMU} onChange={handleCodecChange} name={Opus + PCMU} />}
-                label="Opus, PCMU"
+                label="opus, pcmu"
```

Comment:
Yes. This is to reflect the actual values that devs should use when passing codecPreferences as mentioned to our docs [here](https://www.twilio.com/docs/voice/client/javascript/device#setup) and [here](https://www.twilio.com/docs/voice/client/javascript/device#runpreflight)

Reply:
I see. So the user selects "pcmu", and then "pcmu" is passed to the test.

But this isn't how the Edge Locations work in the same menu. If a user selects "Sao Paulo", then "sao-paulo" is passed to the SDK. 

I still feel that the names of the Codecs should be capitalized so that it is consistent with the names of the Edge Locations. 

![image](https://user-images.githubusercontent.com/12685223/100170605-42213c80-2e83-11eb-9ad2-43d7d5285a1b.png)

Reply:
Good point on the edge locations. I updated the codecs back for consistency.